### PR TITLE
Two little tests for sparse arrays

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

File: stdlib/SparseArrays/test/sparse.jl [hunk 1]
```diff
@@ -82,6 +82,8 @@ end
         SparseMatrixCSC(6, 6, big.([1,2,4,7,8,9,9]), big.([1,1,2,1,2,3,4,5]), big.([1,2,3,4,5,6,7,8])))
     @test sparse(Any[1,2,3], Any[1,2,3], Any[1,1,1]) == sparse([1,2,3], [1,2,3], [1,1,1])
     @test sparse(Any[1,2,3], Any[1,2,3], Any[1,1,1], 5, 4) == sparse([1,2,3], [1,2,3], [1,1,1], 5, 4)
+    # with combine
+    @test sparse([1, 1, 2, 2, 2], [1, 2, 1, 2, 2], 1.0, 2, 2, +) == sparse([1, 1, 2, 2], [1, 2, 1, 2], [1.0, 1.0, 1.0, 2.0], 2, 2)
```
Comment (dkarrasch) [inline stdlib/SparseArrays/test/sparse.jl]: That's okay, but it feels more like testing whether `+` is the default. How about testing a non-default operation?
```julia
@test sparse([1, 1, 2, 2, 2], [1, 2, 1, 2, 2], 2, 2, 2, *) == sparse([1, 1, 2, 2], [1, 2, 1, 2], [2, 2, 2, 4], 2, 2)
```

Comment (kshyatt) [reply stdlib/SparseArrays/test/sparse.jl]: I can add that!

Comment (kshyatt) [reply stdlib/SparseArrays/test/sparse.jl]: Or something similar with `-1`...

Comment (dkarrasch) [reply stdlib/SparseArrays/test/sparse.jl]: Yeah, anything that would take an independent path from the default `combine=+` operation.

**Comment by dkarrasch** [comment]
Offtopic: where do you find coverage test results nowadays? They used to display on the PR's github page (sometimes), but I haven't come across anyone recently.

**Comment by kshyatt** [comment]
I just moseyed on over to the main codecov page, the badge on README worked.

**Comment by kshyatt** [comment]
This good to go?

**Pull request merged**